c/parallel/src/kernels/operators.cpp

```diff
@@ -114,18 +114,19 @@ struct op_wrapper {
 )XXX";
 
   constexpr std::string_view stateful_op = R"XXX(
-struct __align__(OP_ALIGNMENT) op_state {
+struct __align__(OP_ALIGNMENT) op_state {{
```

Comment:
If the opening curly bracket was changed to double bracket here, perhaps it should apply to closing brackets as well as to other opening brackets in `stateful_op` string view.

Reply:
Addressing this comment (and other c.parallel related comments) in the radix sort c.parallel PR https://github.com/NVIDIA/cccl/pull/4350. I will merge those changes back here